portfolio/src/main/java/com/google/sps/servlets/servlets/OAuthServlet.java

```diff
@@ -52,10 +52,10 @@
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
 
     //add credentials to datastore (just do this once for deployment)
-    ReadProperties properties = new ReadProperties("config.properties");
-    DatastoreRetrieval.addEntityToDatastore("Settings", "CLIENT_ID", properties.getProp("clientId"));
-    DatastoreRetrieval.addEntityToDatastore("Settings", "CLIENT_SECRET", properties.getProp("clientSecret"));
-    DatastoreRetrieval.addEntityToDatastore("Settings", "DEVELOPER_TOKEN", properties.getProp("refreshToken"));
+    //ReadProperties properties = new ReadProperties("a.properties");
```

Comment:
remove commented code

Reply:
has been refactored / redesigned in another branch